src/Command/TranskribusAuthCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class TranskribusAuthCommand extends Command
+{
+
+    private $client;
+
+    public function __construct(HttpClientInterface $client)
+    {
+        parent::__construct();
+        $this->client = $client;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setHelp('This command allows you to retrieve and store an access token for testing the Trankribus engine');
```

Comment:
It's not just for testing. Suggest: "…an access token for the Transkribus API."

Also, this should use `setDescription()` instead of `setHelp()`, so this is displayed in the command list and also at the top of the help output. Help is for longer information and appears at the end of the help output.